### ISSUE TITLE
Mobile Chrome triggers animations only after scroll stop

### ISSUE THREAD

**Issue by micrdy** [matthieua/WOW#158]
On desktop and mobile safari all works nice but not on mobile chrome. I think chrome does render the scrolling different?

When I scroll down, the content blocks should appear with a fade effect. But instead, it appears only if the scrolling has completely stopped. In mobile Safari the effect shows up, when the block is entering the viewport. This is correct.

Is there any workaround for showing the animations on mobile chrome like on mobile safari?

**Comment by lancebeaudry** [comment]
@micrdy Have you been able to find a fix for this? I am having the same issue with no luck on a fix.

Thanks

**Comment by micrdy** [comment]
No,  unfortunately not. If have disabled the animation on mobile.
